.circleci/config.yml

```diff
@@ -602,6 +602,11 @@ jobs:
       - run: docker pull alpine:latest # make sure it's fresh
       - deploy:
           command: |
+            git diff go.mod
+            git update-index --assume-unchanged go.mod
```

Comment:
why does this need to be in the ci? seems like a potential security issue

Reply:
I'll update this with a 'run git diff before release' version (instead of ignoring those changes)